windows/ReactNativeWebView/ReactWebView2.cpp

```diff
@@ -85,6 +89,25 @@ namespace winrt::ReactNativeWebView::implementation {
                 WriteWebViewNavigationEventArg(webView, eventDataWriter);
                 eventDataWriter.WriteObjectEnd();
             });
+
+                
+        if (m_messagingEnabled) {
+            m_messageToken = webView.WebMessageReceived([this](auto const& sender, winrt::CoreWebView2WebMessageReceivedEventArgs const& args)
```

Comment:
looks like the sender param is unused; this causes a warning "unreferenced formal parameter"; either remove the name "sender", or write it as `auto const& /* sender */, winrt::CoreWebView2WebMessageReceivedEventArgs const&`